Makefile

```diff
@@ -137,7 +137,15 @@ go-vet:
 
 .PHONY: go-lint
 go-lint: cmd/cli/chart.tgz pkg/envoy/lds/stats.wasm
-	go run github.com/golangci/golangci-lint/cmd/golangci-lint run --config .golangci.yml
+	@if ! golangci-lint version 2>&1 | grep 1.30; then \
+		echo Installing golangci-lint; \
+		go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.30; \
```

Comment:
If CI can use the latest because the version is not hardcoded, why not keep the same behavior for local checks? It would be good to avoid constantly updating this version hardcoded here.

Reply:
CI does have this [hardcoded](https://github.com/openservicemesh/osm/blob/main/.github/workflows/main.yml#L52). I don't necessarily have a problem with periodic updating of version numbers, but the fact that there are multiple sources of truth for the golangci-lint versioning is what I think isn't great and could probably use some more thinking.

Reply:
And I think what we don't want to do is have some sliding window label like `latest`, because my assumptions would be that `latest` for the GitHub Actions component and upstream are probably different. So then we are accepting a bit of version drift there between local and CI.

Reply:
I agree it is better to use the same version. I've never had to run this check locally because my IDE uses golangci-lint as the core linter, and it keeps updating it to the latest version anyway.